apps/condo/domains/common/components/PhoneInput.tsx

```diff
@@ -17,11 +27,25 @@ type PhoneInputRef = {
     numberInputRef: {
         focus: () => void,
     } & ComponentProps<'input'>,
-} 
+}
+
+const getPhoneInputStyles = (style, size: SizeType) => {
+    let height = '32px'
+
+    if (size === 'large') {
+        height = '40px'
+    } else if (size === 'small') {
+        height = '22px'
+    }
+
+    return { ...style, height }
+}
```

Comment:
As for me, this looks simpler:

```suggestion
const getPhoneInputStyles = (style, size: SizeType) => ({
    ...style,
    height: {
        large: '40px',
        small: '22px',
    }[size] || '32px',
})
```